src/qmapshack/gis/rte/router/CRouterRoutino.cpp

```diff
@@ -29,9 +29,8 @@
 #include <QtWidgets>
 #include <routino.h>
 
-
 QPointer<CProgressDialog> CRouterRoutino::progress;
-
+QString currentPrefix = QString("");
```

Comment:
Why is this a variable outside of the class?

Reply:
Testing - moved into class definition.